trinity/console.py

```diff
@@ -56,9 +57,17 @@ def console(ipc_path: Path,
     Method that starts the chain, setups the trinity CLI and register the
     cleanup function.
     """
+    logger = logging.getLogger('trinity')
     if env is None:
         env = {}
 
+    # if ipc_path is not found, print a useful error message
+    if not Path(ipc_path).exists():
+        logger.error("The IPC path at %s is not found. "
```

Comment:
@glaksmono if you build this error message from a function e.g. `create_missing_ipc_error_message(ipc_path: Path) -> str:` then you could import that function in your tests and ensure we don't have to maintain that string as a duplicate in the test.

Reply:
On the other hand, if that method started returning `""` then the test would still pass. I kind of like the style of just checking for one or two key words in the error during the test. (not opposed to extracting the message to a method, btw, just think there are risks to using the same method to generate as to test)

Reply:
I see your point but an empty string seems to be the only return value that would imply that risk and it seems unlikely such a change would ever slip through a review. If in doubt, I'm usually more on the DRY side. No strong opinion though.